core/domain/html_validation_service.py

```diff
@@ -887,23 +887,28 @@ def get_invalid_svg_tags_and_attrs(svg_string):
         svg_string: str. The SVG string.
 
     Returns:
-        tuple. A 2-tuple, the first element of which is a list of invalid tags,
-        and the second element of which is a list of invalid attributes.
+        tuple(list(str), list(str)). A 2-tuple, the first element of which
```

Comment:
Just a thought -- do you want to put a size limit on SVG length (at the controller level) and reject "too big" SVGs in order to prevent attacks where the checker/validator gets overloaded and times out? Might be prudent. We're also asking contributors to ensure SVGs are < 100 KB anyway so might be worth enforcing that.

Reply:
Yep, I'll do that. Should we also have a restriction on the UI for the SVG size?

Reply:
Yes, let's mention it. Thanks!